web/src/app.dev.js

```diff
@@ -22,7 +22,15 @@ const history = createHistory();
 
 const middleware = [thunk, routerMiddleware(history), createLogger()];
 
-const store = createStore(reducer(history), applyMiddleware(...middleware));
+/* eslint-disable no-underscore-dangle */
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+/* eslint-enable */
```

Comment:
If you don't want to commit this to `app.dev.js`, that's fine by me -- but, man, Redux devtools make things easier!!!

Reply:
You can thank Mr. Heather for that one -- dinner conversation was me complaining about not being able to get my head around what the state store looks like. This is lifechanging for me:
<img width="716" alt="Screen Shot 2019-07-26 at 9 39 10 AM" src="https://user-images.githubusercontent.com/509309/61963644-543a5480-af89-11e9-8773-ec70e8952fae.png">